x/examples/outline-connectivity-app/shared_frontend/pages/connectivity_test/localize.json

```
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://raw.githubusercontent.com/lit/lit/main/packages/localize-tools/config.schema.json",
+  "sourceLocale": "en",
+  "targetLocales": ["es-419", "zh-Hans", "fa-IR"],
```

I don't think you need to specify a country for Farsi.
```suggestion
  "targetLocales": ["es-419", "zh-Hans", "fa"],
```